Services/Administration/OJS.Services.Administration.Data/Implementations/ContestsDataService.cs

```diff
@@ -1,27 +1,33 @@
 namespace OJS.Services.Administration.Data.Implementations
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
     using Microsoft.EntityFrameworkCore;
     using OJS.Common.Enumerations;
     using OJS.Data;
     using OJS.Data.Models.Contests;
     using OJS.Data.Models.Problems;
+    using OJS.Services.Common;
     using OJS.Services.Common.Data.Implementations;
+    using OJS.Services.Common.Models.Contests;
     using OJS.Services.Infrastructure;
     using SoftUni.AutoMapper.Infrastructure.Extensions;
     using SoftUni.AutoMapper.Infrastructure.Models;
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Linq.Expressions;
-    using System.Threading.Tasks;
 
     public class ContestsDataService : DataService<Contest>, IContestsDataService
     {
         private readonly IDatesService dates;
+        private readonly IContestsActivityService activityService;
 
-        public ContestsDataService(OjsDbContext db, IDatesService dates)
+        public ContestsDataService(OjsDbContext db, IDatesService dates, IContestsActivityService activityService)
```

Comment:
Please follow the convention and add each class/interface to a new row. When we have multiple items in the constructor, this will create a large horizontal row.